templates/keymanager/resources/vkcs_keymanager_container.md.tmpl

```diff
@@ -12,13 +12,9 @@ description: |-
 
 ## Example Usage
 ### Simple secret
```

Comment:
We do not need sub header for the only example

Reply:
yes, removed